lib/order.rb

```diff
@@ -0,0 +1,49 @@
+require 'dotenv'
+Dotenv.load('./twilio_credentials.env')
+require_relative 'list_of_dishes'
+require_relative 'text_handler'
+
+class Order
+  attr_reader :summary
+
+  def initialize(list_of_dishes, selections, total, phone_number = ENV['ORDER_PHONE_NUMBER'])
+    raise ArgumentError, 'order must not be empty' if selections.empty?
+
+    @phone_number = phone_number
+    populate_summary(list_of_dishes, selections)
+    verify_total(total)
+  end
+
+  def send_text(text_handler = TextHandler.new)
+    text_handler.send_confirmation_text(@phone_number)
+  end
+
+  private
+
+  def populate_summary(list_of_dishes, selections)
+    @summary = {}
+
+    selections.each do |dish, quantity|
+      raise ArgumentError, 'ordered dish not on menu' if list_of_dishes.get_dish(dish).nil?
```

Comment:
I wonder if we could avoid a `nil` check here. Could `Order` ask `ListOfDishes` if it has a dish?